index.html

```diff
@@ -4264,6 +4264,48 @@ <h3>Spectrum of Privacy</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Software Trust Boundaries</h3>
+
+        <p>
+There are a variety of trust boundaries that exist in the
+<a href="#ecosystem-overview">ecosystem described by this specification</a>.
+Just like an individual using a web browser trusts the web browser not to betray
+their trust by uploading their personal information to a data broker, a similar
+trust exists between the roles in the ecosystem described by this specification
+and the software that operates on the behalf of each of those roles. For
+example,
+        </p>
+
+        <ul>
+          <li>
+An <a>issuer</a> trusts the issuer software it uses to not issue <a>verifiable
+credentials</a> without its permission.
+          </li>
+          <li>
+A <a>verifier</a> trusts the verifier software it uses to be truthful regarding
+the verification status of the <a>verifiable credentials</a> and
+<a>verifiable presentations</a> that it is checking.
+          </li>
+          <li>
+A <a>holder</a> trusts the digital wallet software it uses to not divulge
```

Comment:
```suggestion
A <a>holder</a> trusts the digital wallet software it uses not to divulge
```

(Same remark as above.)

Reply:
Fixed in @TallTed's suggestion.